.github/workflows/workflow.yml

```diff
@@ -0,0 +1,44 @@
+name: Continuous Integration 
+on:
+  push:
+    paths-ignore:
+      - README.md
+  pull_request:
+    paths-ignore:
+      - README.md 
+  schedule:
+    - cron: "0 23 * * *"
+
+jobs:
+  build:
+    name: Build and test 
+    runs-on: ubuntu-latest
+    
+    steps: 
+    - uses: actions/checkout@v2 
+    - name: build test docker image
+      run: docker build --target test --tag todo_app:test .
+
+    - name: run docker integration test container
+      run: docker run --env-file .env.test todo_app:test todo_app/tests/test_app.py
```

Comment:
Similar comments to the other two run commands - is this a sustainable pattern? At the moment every single test file will need it's own CI entry, which might cause you a _lot_ of pain as you start to manage more test files (think handling 100+ files!)

What might be nicer would be to switch the e2e test into its own `tests_e2e` folder or similar, and running the folders separately, e.g.
```
docker run --env-file .env.test todo_app:test todo_app/tests
```
&
```
docker run -e TRELLO_API_KEY=${{secrets.TRELLO_API_KEY}} -e TRELLO_API_TOKEN=${{secrets.TRELLO_API_TOKEN}} -e SECRET_KEY=anything todo_app:test todo_app/tests_e2e/
```